ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/preferences/PDECompilersConfigurationBlock.java

```diff
@@ -933,15 +933,15 @@ private void addBuilder(Control control) {
 				continue;
 			}
 			if (controls.contains(control)) {
-				switch (key.intValue()) {
+				switch (key.intValue())	{
```

Comment:
If you format the switches completely it reformats all of them. May be an error: configured is 
```
switch {
}
```
but result ist
```
switch 
  {
  }
```

An "}" deeper then the "{" looks like an error and is highly missleading. therefore i only removed the ";" without reformatting.

Reply:
That's strange. For me the result is always the braces on the next line. Sounds like a bug in the JDT formatter, if it is configured to be on the same line as the keyword.


> An "}" deeper then the "{" looks like an error and is highly missleading. therefore i only removed the ";" without reformatting.

Yes, that's good. But in this case you inserted a tab-stop that has, at least for me, the same length like a space.

Reply:
In this case i auto-formatted the whole method because the indent on the closing brackets was wrong. And as far as i remember manually moved the opening bracket back to the switch line.